utils/gen_config.py

```diff
@@ -0,0 +1,174 @@
+# coding=utf-8
+"""Update scan facts in config file given scan data.
+
+Run this script as a command line tool to use csv with data about hosts and
+profiles and report file from rho scan to generate camayoc config file.
+
+.. note::
+
+    This tool reads in your camayoc config file in $XDG_CONFIG_HOME and outputs
+    it to the $CWD with new data. It discards old hosts data but uses auth,
+    profile, and any other data provided in your $XDG_CONFIG_HOME config file
+    (if it exists). This does not modify your $XDG_CONFIG_HOME config file
+    directly, that is left to you. Always backup your old config file before
+    you replace it with the generated one!
+
+Example:
+    .. code-block:: bash
+
+        python gen_config.py --profilecsv profiles.csv --reportcsv report.csv
+"""
+from camayoc import config
```

Comment:
Please organize imports as follow:

First Python standard library imports
Second third party libraries import
Third own library imports (camayoc)

Each of the above group should follow the following and a blank line can be used to separate the groups.

First lines starting with import (ordered in alphabetical order)
Then lines start with from ... import ... (ordered in alphabetical order)